Fix hardcoded paths in executable scripts of PROMALS3D
author: Alex Domingo (Vrije Universiteit Brussel)
diff --git a/src/progress_for_web.py.orig b/src/progress_for_web.py
index bdac224..d913f17 100644
--- a/src/progress_for_web.py.orig
+++ b/src/progress_for_web.py
@@ -397,7 +397,7 @@ def get_csv_string(alnfile, caa_freq):
                         except:
                                 pass
 
-	command = "/home/jpei/t2/bar/promals_package/bin/al2co_consensus -i %s -t %s.csv.aln -g 0.25 -b 1000000 -consensus T -freq_cutoff %f > /dev/null" %(alnfile, alnfile, caa_freq)
+	command = "al2co_consensus -i %s -t %s.csv.aln -g 0.25 -b 1000000 -consensus T -freq_cutoff %f > /dev/null" %(alnfile, alnfile, caa_freq)
         #print command
 	#command = "/home/jpei/t2/bar/promals_package/bin/al2co -i %s -t /tmp/csv.aln -g 0.25 -b 1000000 > /dev/null" %(alnfile)
 	os.system(command)
@@ -418,7 +418,7 @@ def get_csv_string(alnfile, caa_freq):
 
 def run_promals(inputfile):
 	
-	command = "/home/jpei/profile_hmm/profile_hmm/progress %s -outfile %s.promals.aln > %s.log" %(inputfile, inputfile, inputfile)
+	command = "progress %s -outfile %s.promals.aln > %s.log" %(inputfile, inputfile, inputfile)
 	os.system(command)
 
 	if(not os.path.isfile(inputfile+".promals.aln") ):
@@ -473,7 +473,7 @@ if __name__ == "__main__":
 
 	#############################
 	'''
-	ref_file = "/home/jpei/prefab4/ref/" + inputfile
+	ref_file = inputfile
 	reflines = open(ref_file).readlines()
 	for line in reflines: 
 		tmpline = ""
