--- bin/fusioncatcher.py	2019-04-04 14:56:27.000000000 +0200
+++ bin/fusioncatcher.py	2019-07-03 22:51:46.512772571 +0200
@@ -2253,7 +2253,7 @@
     # check version
     os.system(_BP_+"bbmap.sh --version 2>&1 |head -2 |tail -1 > '%s'" % (outdir('bbmap_version.txt'),))
     last_line = file(outdir('bbmap_version.txt'),'r').readline().lower().rstrip("\r\n")
-    correct_version = ('bbmap version 38.44',)
+    correct_version = ('bbmap version 38.56',)
     if last_line not in correct_version:
         job.close()
         os.system("which bbmap.sh > '%s'" % (outdir('bbmap_path.txt'),))
@@ -2457,7 +2457,7 @@
 
         os.system(_SR_+"STAR --version > '%s'" % (outdir('star_version.txt'),))
         last_line = file(outdir('star_version.txt'),'r').readline().lower().rstrip("\r\n")
-        correct_version = '2.7.0f'
+        correct_version = '2.7.1a'
         #correct_version = 'star_2.5.4b'
         #correct_version = 'star_2.5.2b'
         #correct_version = 'star_2.5.2a'
